projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl

```diff
@@ -241,5 +241,5 @@ Image Pull Secret in json format
 {{- end }}
 
 {{- define "pipelinesControlServicePullSecretJson" }}
-    {{ include "buildImagePullSecretJson" (list (include "pipelines-control-service.image" .) .Values.image.registryUsernameReadOnly .Values.image.registryPasswordReadOnly) }}
+    {{ include "buildImagePullSecretJson" (list .Values.image.registry .Values.image.registryUsernameReadOnly .Values.image.registryPasswordReadOnly) }}
```

Comment:
isn't that declared in helpers https://github.com/vmware/versatile-data-kit/blob/1dbd84688f07844ee51189e831f6cbd314b2b5ab/projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl#L72

Reply:
Im not using that one anymore.

Reply:
but even if it is why is that relevant?

Reply:
I saw you are deleting it. Why are you deleting it? It still seems defined in `_helpers.tpl` after your change? 
Is that change needed, isn't the `list (include "pipelines-control-service.image" .)` actually enlisting all the `pipelines-control-service.image` contained sub values, that already include `.Values.image.registry` as well?
```
{{/*
Return the proper Control Service image name
*/}}
{{- define "pipelines-control-service.image" -}}
{{- $globalRegistryOverrideName := .Values.image.globalRegistryOverride -}}
{{- $registryName := .Values.image.registry -}}
{{- $repositoryName := .Values.image.repository -}}
{{- $tag := .Values.image.tag | toString -}}
{{/*
```